test/forms/ConvertedPackagingCreditFormProviderSpec.scala

```diff
@@ -68,5 +69,12 @@ class ConvertedPackagingCreditFormProviderSpec extends DecimalFieldBehaviours {
                                    FormError(fieldName, "convertedPackagingCredit.error.required")
       )
     }
+
+    "must reject greater than the allowed amount" in {
```

Comment:
this test is actually a duplicate of the scaffold ones however i think it adds variation that this is specific behaviour